timescaledb/tutorials/grafana/visualizations/histograms.md

```diff
@@ -0,0 +1,269 @@
+To get the desired effect we need to change our symbol variable from a text variable to a query variable with the multi-value option enabled.
+
+<procedure>
+
+1.  We use this query to fetch all companies in our dataset:
+
+    ```sql
+    SELECT DISTINCT symbol FROM company ORDER BY asc;
+    ```
+
+2.  Create a new dashboard variable with the previous query: 
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/symbol_variable_query.png" alt="symbol variable query"/>
+    If we select more than one value, TimescaleDB returns the transactions for 
+    all selected values. Grafana buckets those in separate histograms.
+  
+3.  Update our main query to use the following query:  
+
+    ```sql
+    SELECT time_bucket('$bucket_interval',time) AS time,
+        symbol,
+        AVG(price) AS avg_price
+    FROM stocks_real_time srt
+    WHERE symbol in ($symbol)
+        AND time >= $__timeFrom()::timestamptz AND time < $__timeTo()::timestamptz
+    GROUP BY time_bucket('$bucket_interval',time), symbol;
+    ```
+
+4.  When we visualize this query, we get the following histogram:
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/multiple_histograms.png" alt="multiple histograms"/>
+
+    We can clearly see the 3 distinct histograms but it's impossible to tell 
+    them apart from each other. 
+    
+5.  Click on the green line left of the legend and pick a color:
+
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/color_options.png" alt="color options"/>
+
+6.  Doing so results in the following panel:
+    
+    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/colored_in_histogram.png" alt="colored in histogram"/>
+
+    Here we can clearly see the 3 different price distributions for each stock.
+
+</procedure>
+
+## Price/Volume Histogram
+
+An interesting metric to investigate is the trade volume distribution. This 
+distribution gives us insight into the volume at which people buy a stock. 
+The dataset has a column with the daily cumulative volume which we can use to 
+determine the individual transaction volume. To calculate the volume of data for
+each bucket, we need to find the maximum `day_volume` value for the symbol and 
+then subtract each value from the previous bucket maximum value. The difference 
+of these two rows produces the volume within the bucket of time. We can 
+accomplish this with a slightly more complex query that pre-aggregates our data 
+using the TimescaleDB `time_bucket()` hyperfunction and the standard PostgreSQL 
+`MAX()` function, and then use the PostgresSQL `LAG()` window function to 
+subtract each row from the previous, ordered by `time` descending.
```

Comment:
```suggestion
The `stocks_real_time` hypertable contains a column with the daily cumulative
volume. You can use this to calculate the volume of data for each bucket. First,
find the maximum `day_volume` value for a symbol within a bucket. Then subtract
each maximum from the previous bucket's maximum. The difference equals the
volume for that bucket.

You can do this with a pre-aggregation query, using:
*   TimescaleDB's `time_bucket` function.
*   PostgreSQL's `max` function.
*   PostgreSQL's `lag` function. Use this to subtract each from from the previous,
    when the rows are ordered by descending `time`.
```
Is descending correct here?

Also suggest linking to API references for these functions